raiden/tests/integration/fixtures/blockchain.py

```diff
@@ -17,6 +21,8 @@
 # pylint: disable=redefined-outer-name,too-many-arguments,unused-argument,too-many-locals
 
 _GETH_LOGDIR = os.environ.get('RAIDEN_TESTS_GETH_LOGSDIR', False)
+_GETH_DATADIR = os.environ.get('RAIDEN_TESTS_GETH_DATADIR', False)
```

Comment:
hmm, I don't see why the default return value `False` would be necessary -- `None` should be fine for all later `if OS_VAR:...` conditionals. Can you remove it for both `_GETH_...` variables?

Reply:
I just realized that the DATADIR variables are not used anymore anyway, so I removed them (and the `False` default for `_GETH_LOGDIR`).

Reply:
:+1: